site/docs/master/aws-config.md

```diff
@@ -265,15 +274,17 @@ cat > velero-policy.json <<EOF
     ]
 }
 EOF
-```
-```bash
+    ```
+    ```bash
 aws iam put-role-policy \
   --role-name velero \
   --policy-name velero-policy \
   --policy-document file://./velero-policy.json
-```
+    ```
+
 4. Update `AWS_ACCOUNT_ID` & `VELERO_ROLE_NAME` with `kubectl edit deploy/velero -n velero` and add the following annotation:
-```
+
+    ```
 ---
```

Comment:
This doesn't render properly in GH, I believe the content needs to be indented to line up with the backticks.